website/src/components/Footer/index.tsx

```diff
@@ -136,55 +136,93 @@ const Footer: React.FC = () => {
     //     );
     // };
 
+
+
+    const  getdatesForContribution = () => {
+        var today = new Date()
+        var priorDate = new Date();
+        priorDate.setDate(priorDate.getDate() - 30);
+        return {
+            today,
+            priorDate,
+            todayTimestamp: today.valueOf(),
+            priorDateTimestamp: priorDate.valueOf()
+        };
+    }
     const DisplayTopContributors: React.FC = () => {
         const githubApiContributors = API.GITHUB_CONTRIBUTORS;
         const [isLoaded, setIsLoaded] = useState<boolean>(false);
         const [items, setItems] = useState([]);
+        const reqBodyToFetchContributors = {
+            "queries": [
+                {
+                    "refId": "A",
+                    "datasourceId": 1,
+                    "rawSql": "select\n  row_number() over (order by value desc) as \"Rank\",\n  name,\n  value\nfrom\n  shpr_auth\nwhere\n  series = 'hpr_authall'\n  and period = 'm'",
+                    "format": "table",
+                    "intervalMs": 86400000,
+                    "maxDataPoints": 1255
+                }
+            ],
+            "range": {
+                "from": `${getdatesForContribution().priorDate}`,
+                "to": `${getdatesForContribution().today}`,
+                "raw": {
+                    "from": "now-1m",
+                    "to": "now"
+                }
+            },
+            "from": `${getdatesForContribution().priorDateTimestamp}`,
+            "to": `${getdatesForContribution().todayTimestamp}`
+        }
+        
         //getting the top contributors from github by sending the api order as desc
-
         useEffect(() => {
-           fetch(githubApiContributors)
-           .then((res) => res?.json())
-           .then(
-             (result) => {
-               setIsLoaded(true);
-               setItems(result);
-             },
-             (error) => {
-               setIsLoaded(true);
-               console.error(error);
-             }
-           );
-          return () => {
-            setItems([]);
-          };
-        }, []); // eslint-disable-line react-hooks/exhaustive-deps
+            fetch(githubApiContributors, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify(reqBodyToFetchContributors),
+            })
+                .then(response => response.json())
+                .then(contributors => {
+                    setIsLoaded(true);
+                    setItems(contributors?.results?.A?.frames[0]?.data?.values[1]);
+                })
+                .catch((error) => {
+                    console.error('Error:', error);
+                });
+        });
```

Comment:
if we dont give any dependency array it will send the POST on every render! Please add an empty `[]` to prevent this. Unless we need to fetch it on every render